Panic in `engine.sanitizeUpdatecliManifestFilePath`

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

I ran `updatecli` with an incorrect path for `--config`. This resulted in the following Go panic:

```
updatecli diff --config ./updatecli/updatecli.d --values updatecli/values.d/updatecli.yaml


+++++++++++
+ PREPARE +
+++++++++++

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x68bf085]

goroutine 1 [running]:
github.com/updatecli/updatecli/pkg/core/engine.sanitizeUpdatecliManifestFilePath({0xc000574cd0?, 0xc000c0f478?, 0x3a60b9f?})
	/home/runner/work/updatecli/updatecli/pkg/core/engine/utils.go:22 +0xc5
github.com/updatecli/updatecli/pkg/core/engine.(*Engine).LoadConfigurations(0xa2ef7a0)
	/home/runner/work/updatecli/updatecli/pkg/core/engine/configuration.go:60 +0x1cf
github.com/updatecli/updatecli/pkg/core/engine.(*Engine).Prepare(0xa2ef7a0)
	/home/runner/work/updatecli/updatecli/pkg/core/engine/prepare.go:23 +0x46
github.com/updatecli/updatecli/cmd.run({0x6937175?, 0x0?})
	/home/runner/work/updatecli/updatecli/cmd/root.go:126 +0x4a7
github.com/updatecli/updatecli/cmd.init.func9(0xa2a1180?, {0xc000b6bc40?, 0x0?, 0x4?})
	/home/runner/work/updatecli/updatecli/cmd/diff.go:38 +0x247
github.com/spf13/cobra.(*Command).execute(0xa2a1180, {0xc000b6bc00, 0x4, 0x4})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.9.1/command.go:1019 +0xa91
github.com/spf13/cobra.(*Command).ExecuteC(0xa29fb80)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.9.1/command.go:1148 +0x46f
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.9.1/command.go:1071
github.com/updatecli/updatecli/cmd.Execute()
	/home/runner/work/updatecli/updatecli/cmd/root.go:50 +0x5b
main.main()
	/home/runner/work/updatecli/updatecli/main.go:8 +0xf
```

The path `./updatecli/updatecli.d` doesn't exist in my repository.

### Expected Behavior

A more clean error message.

### Steps To Reproduce

_No response_

### Environment

```markdown
- OS: macOS 15.5
- updatecli: 0.103.0
```

### Pipeline Graph

_No response_

### Anything else?

_No response_

--- a/pkg/core/engine/utils.go
+++ b/pkg/core/engine/utils.go
@@ -16,6 +16,10 @@ import (
 func sanitizeUpdatecliManifestFilePath(rawFilePaths []string) (sanitizedFilePaths, sanitizedPartialPaths []string) {
 	for _, rawFilePath := range rawFilePaths {
 		rawFileInfo, err := os.Stat(rawFilePath)
+		if err != nil {
+			logrus.Error(fmt.Sprintf("Loading Updatecli manifest %q: %s", rawFilePath, err))
+			continue
+		}
 
 		// If the manifest if a directory, then we walk trough it to find all manifest files
 		// and partial files.

